NEWS

```diff
@@ -69,6 +69,37 @@ New Functionality
   Given this is the first iteration of this feature, feedback around usability and
   use-cases that aren't covered are more than welcome.
 
+- The event keyword in signatures was extended to support choosing a custom event
+  to raise instead of ``signature_match()``. This can be more efficient in certain
+  scenarios compared to funneling every match through a single event.
+
+  The new syntax is to put the name of the event in brackets before the string
+  or identifier used as message. As an extension, it is possible to only provide
+  the bracketed event name. In this case, the framework expects the event's
+  parameters to consist of only state and data as follows:
+
+	signature only-event {
+		payload /.*root/
+		event [found_root]
+	}
+
+	event found_root(state: signature_state, data: string) { }
+
+  Passing an additional message parameter to a custom event is possible with the
+  following syntax. The custom event's parameters need to align with those for the
+  ``signature_match()` event:
+
+	signature event-with-msg {
+		payload /.*root/
+		event [found_root_with_msg] "the-message"
```

Comment:
Perhaps remind the reader that `"the-message"` can instead be a string-valued Zeek expression (or just a variable?). I was initially surprised at the `message_as_id` BTest because of this